lib/puppet/provider/exec/powershell.rb

```diff
@@ -86,7 +90,7 @@ def run(command, check = false)
 
       Puppet.debug "STDERR: #{result[:errormessage]}" unless result[:errormessage].nil?
 
-      output = Puppet::Util::Execution::ProcessOutput.new(stdout.to_s || '', exit_code)
+      output = Puppet::Util::Execution::ProcessOutput.new(stdout.to_s + native_out.to_s, exit_code)
```

Comment:
* `native_out` could be `nil`
* `stdout.to_s` was already in the code ... I think it too can be `nil` IIRC, but feel free to verify.